fdbclient/ServerKnobs.cpp

```diff
@@ -623,6 +623,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	// Block cache key-value checksum. Checksum is validated during read, so has non-trivial impact on read performance.
 	init( ROCKSDB_BLOCK_PROTECTION_BYTES_PER_KEY,                  0 ); if ( randomize && BUGGIFY ) ROCKSDB_BLOCK_PROTECTION_BYTES_PER_KEY = 8; // Default: 0 (disabled). Supported values: 0, 1, 2, 4, 8.
 	init( ROCKSDB_ENABLE_NONDETERMINISM,                      false );
+	init( SHARDED_ROCKSDB_ALLOW_MULTIPLE_RANGES,              false );
```

Comment:
Do we want to enable it in simulation?

Reply:
It's enabled in unit test only.

Reply:
I'd be interested in a bit of doc. on what it is that this feature is. Thanks.

Reply:
In short, it relaxes some of the checks we currently do to ensure key ranges are continuous. I’m planning to write a doc explaining how shards are placed in the storage engine. In 99% of cases, key ranges and shards have a 1:1 mapping, but data movement can lead to some corner cases.